app/src/main/java/com/codesoom/assignment/enums/HttpMethod.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.enums;
+
+/**
+ * 프로젝트 내에서 사용되는 HttpMethod를 정의해놓은 Enum입니다.
+ *
+ * @author DevRunner21
+ * @version 1.0
+ * @since 2021.05.07
+ * @see <a href="https://tools.ietf.org/html/rfc7231#section-4">HttpReuest Method(RFC 7231)</a>
+ */

```

Comment:
`enum`에 대해 설명할 때 `이것은 ~~ enum입니다.`라고 이야기할 필요는 없어요.

```suggestion
/**
 * HttpMethod를 정의합니다.
 * @see <a href="https://tools.ietf.org/html/rfc7231#section-4">HttpReuest Method(RFC 7231)</a>
 */
```

`@author`, `@version`, `@since` 등도 git으로 코드를 관리하고 있기 때문에 특수한 경우가 아니라면 굳이 작성할 필요는 없습니다.